databuilder/databuilder/extractor/es_base_extractor.py

```diff
@@ -0,0 +1,86 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+import abc
+from typing import (
+    Any, Dict, Iterator, Optional, Union,
+)
+
+from pyhocon import ConfigTree
+
+from databuilder.extractor.base_extractor import Extractor
+
+
+class ElasticsearchBaseExtractor(Extractor):
+    """
+    Extractor to extract index metadata from Elasticsearch
+    """
+
+    ELASTICSEARCH_CLIENT_CONFIG_KEY = 'client'
+    ELASTICSEARCH_EXTRACT_TECHNICAL_DETAILS = 'extract_technical_details'
```

Comment:
could you give a comment on config value ?

Reply:
it's included it in docs.

Reply:
As for dbapi - from the docs I can see that `This library supports Elasticsearch 7.X versions.` - I tried it with 6.8.13 and failed. Also - I am not sure the column statistics part would be achieved using this library.